src/channels.rs

```diff
@@ -240,6 +326,34 @@ impl Course {
         guild.delete_role(&ctx.http, self.role).await?;
         Ok(())
     }
+
+    async fn rename_course(
```

Comment:
maybe rename this (heh) to just `rename`? it's redundant since it's already a method on `course`. I also noticed other methods in `Course` also have this problem, can you also fix those?